crmsh/ui_cluster.py

```diff
@@ -190,25 +190,36 @@
         for node in node_list:
             logger.info("The cluster stack started on {}".format(node))
 
-    @command.skill_level('administrator')
-    def do_stop(self, context, *args):
-        '''
-        Stops the cluster stack on all nodes or specific node(s)
-        '''
-        node_list = parse_option_for_nodes(context, *args)
-        for node in node_list[:]:
-            if not utils.service_is_active("corosync.service", remote_addr=node):
-                if utils.service_is_active("sbd.service", remote_addr=node):
-                    utils.stop_service("corosync", remote_addr=node)
-                    logger.info("The cluster stack stopped on {}".format(node))
-                else:
-                    logger.info("The cluster stack already stopped on {}".format(node))
-                node_list.remove(node)
-            elif not utils.service_is_active("pacemaker.service", remote_addr=node):
+    @staticmethod
+    def _stop_partial_service(node):
+        """
+        Stops partial cluster service
+        """
```

Comment:
Can you rename this function or change the docstring to make the meaning of return value more clear?